methods.tex

```diff
@@ -284,6 +280,9 @@ \subsection{Material Flow}
 uranium was overlooked. However, further in the future where the depleted
 uranium inventory drains, reprocessed uranium (or, natural uranium) will need to be utilized. 
 
+The simulated fuel cycle is illustrated in figure \ref{diag:fc}.
```

Comment:
Ideally, this shouldn't be all by itself. Perhaps something substantive can be said?